defaults/main.yml

```diff
@@ -13,5 +13,17 @@ letsencrypt_certbot_verbose: true
 letsencrypt_pause_services: []
 # Force Certificate Reneval
 letsencrypt_force_renew: true
-# Additional certbot arguments
-letsencrypt_certbot_args: []
+# certbot required parameters
+letsencrypt_certbot_default_args:
+  - "certonly"
+  - "--text"
+  - "-n"
+  - "--no-self-upgrade"
+  - "--domains {{ _letsencrypt_domains | default(letsencrypt_domain) }}"
+  - "--agree-tos"
+
+
+# certbot custom arguments -- see: https://certbot.eff.org/docs/using.html
+letsencrypt_certbot_args:
+  - "--standalone"
+  - "--expand"
```

Comment:
Maybe the name `letsencrypt_certbot_args` is a bit misleading. It should give the ability to pass additional flags to certbot. In the given case you'd need to also pass `["--standalone", "--expand", "--my-extra-flag"]` instead of just `["--my-extra-flag"]`. This could break upwards compatibility of roles in use passing extra flags, when maintainers did not notice the change or in consequence not allow us to make any further changes. It becomes extra hard when we want to keep support for different versions of Certbot.

We could rename the `letsencrypt_certbot_args` to `letsencrypt_certbot_extra_args` and add two more variables to the configuration
```yaml
letsencrypt_certbot_expand: true
letsencrypt_certbot_mode: "standalone"
# certbot custom arguments -- see: https://certbot.eff.org/docs/using.html
letsencrypt_certbot_extra_args: []
```

and two more tasks to `cert.yml`
```yaml
- set_fact: _letsencrypt_certbot_args="{{ _letsencrypt_certbot_args + ['--standalone'] }}"
  when: letsencrypt_certbot_mode == "standalone"
- set_fact: _letsencrypt_certbot_args="{{ _letsencrypt_certbot_args + ['--expand'] }}"
  when: letsencrypt_certbot_expand == true
```